### ISSUE TITLE
[PARSE] Remove story content from response of search request

### ISSUE THREAD

**Issue by fthdgn** [bounswe/bounswe2015group7#86]
Current search function ("search_story_withTag") returns list of Story objects. The Story objects also contain main content of the stories. This makes the response very large. We should modify the response and remove details. The details of a story will be accesed by the story_get API.( #87)

**Comment by bilalyasar** [comment]
So what will be your returning object?

**Comment by fthdgn** [comment]
```
{
    "result": [
        {
            "__type": "Object",
            "className": "Story",
            "createdAt": "2015-11-05T19:37:11.485Z",
            "lastEditor": {
                "__type": "Pointer",
                "className": "_User",
                "objectId": "N9a00UfgSn"
            },
            "objectId": "RCOAL3Eblf",
            "owner": {
                "__type": "Pointer",
                "className": "_User",
                "objectId": "N9a00UfgSn"
            },
            "tagsArray": [
                "country",
                "anatolia",
                "asia"
            ],
            "title": "turkey",
            "updatedAt": "2015-11-05T19:37:11.485Z"
        },
        {
            "__type": "Object",
            "className": "Story",
            "content": "turkey is a country",
            "createdAt": "2015-11-05T19:38:58.686Z",
            "lastEditor": {
                "__type": "Pointer",
                "className": "_User",
                "objectId": "N9a00UfgSn"
            },
            "objectId": "8QI7K2j84X",
            "owner": {
                "__type": "Pointer",
                "className": "_User",
                "objectId": "N9a00UfgSn"
            },
            "tagsArray": [
                "country",
                "anatolia",
                "asia"
            ],
            "title": "turkey",
            "updatedAt": "2015-11-05T19:38:58.686Z"
        }
    ]
}

```

This is current response. I think removing the "content" is enough.

**Comment by fthdgn** [comment]
Closed via df01596bc44d9dc052c82b7ed3ecd87a5b8859bb
